### PR TITLE
13.1 image augmentation

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

File: chapter_computer-vision/imagine-augmentation.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+* 图像增强基于现有的训练数据生成随机图像，以应对过度拟合。
+* 为了在预测过程中得到确切的结果，我们通常对训练样本只进行图像增强，而在预测过程中不使用随机操作的图像增强。
+* 我们可以从Gluon的 `transforms` 模块中调用图像增强的函数。
+
+## 练习
+
+1. 对模型的训练中不使用图像增强： `train_with_data_aug(no_aug, no_aug)` 。比较使用和不使用图像增强的训练结果和测试精度。这个对比实验能支持图像增强可以减轻过度拟合的论点吗？为什么？
+2. 在基于CIFAR-10数据集的模型训练中加入不同的图像增强方法。观察实施结果。
+3. 参考MXNet文档，在Gluon的 `transforms` 模块中还提供了哪些其他图像增强方法？
```
Comment (goldmermaid) [inline chapter_computer-vision/imagine-augmentation.md:369]: Question: shall we remove the MXNet related questions in exercise or summary?

**Comment by mli** [comment]
Job d2l-zh/PR-770/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-770/

**Comment by mli** [comment]
Job d2l-zh/PR-770/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-770/

**Comment by mli** [comment]
Job d2l-zh/PR-770/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-770/

**Comment by goldmermaid** [comment]
> Thanks! Just to check: is this PR based on the latest version in https://github.com/d2l-ai/d2l-zh/tree/master/chapter_computer-vision ?

Yes! It's the latest and based on the [changes](https://github.com/d2l-ai/d2l-en/commit/685e381e5d631f8d16f28fe8827e388fbd7fea40) you just made!

**Comment by astonzhang** [comment]
You may refer to existing terms used in v1:
http://zh.d2l.ai.s3-website-us-west-2.amazonaws.com/chapter_computer-vision/image-augmentation.html

**Pull request merged**